packages/gateway/cmd/database.ts

```diff
@@ -50,8 +50,8 @@ const _ = (async () => {
     '0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80'
   const rpcURL = process.env.RPC_URL || 'http://localhost:8545'
 
-  const chainID = process.env.CHAIN_ID || '31337'
-  const chain = getChain(parseInt(chainID))
+  const chainID = parseInt(process.env.CHAIN_ID || '31337')
```

Comment:
Good to maintain the same variable name across files. `chainId`

Reply:
the difference is that the client is using a flag-based config
![image](https://github.com/user-attachments/assets/f5e5f4fb-35fe-49da-a69a-105047e9ac21)

the other components gather the variables from the .env file.

I'd have to change how things are being initialized to standardize it